Tasks/PowerShellV2/task.json

```diff
@@ -17,8 +17,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 170,
-        "Patch": 1
+        "Minor": 172,
```

Comment:
Could you please bump task version according to [instruction](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md)